server/src/spacon/db/conn.clj

```diff
@@ -44,17 +44,19 @@
       :min-pool-size     2
       :initial-pool-size 2})))
 
-(defqueries "sql/schema.sql" {:connection db-spec})
-(defn create-spacon-schema [] (create-spacon-schema!))
-
 (defn loadconfig []
   (log/debug "Loading database migration config")
   {:datastore  (jdbc/sql-database db-spec {:migrations-table "spacon.migrations"})
    :migrations (jdbc/load-resources "migrations")})
 
+(defn create-schema []
+  (log/debug "Creating schema if it doesnt exist")
+  (sql/execute! db-spec ["CREATE SCHEMA IF NOT EXISTS spacon"])
+  (repl/migrate (loadconfig)))
```

Comment:
I think the create-schema function should only create the schema and nothing else.  So you should remove this line but add `(create-schema)` to the load config function, so it creates the schema before trying to return a datastore that depends on it.